storage/series_test.go

```diff
@@ -208,6 +207,61 @@ func TestHistogramSeriesToChunks(t *testing.T) {
 		PositiveBuckets: []float64{2, 2, 7, 2},
 	}
 
+	// Gauge histogram
```

Comment:
Nit: We usually end comments with a period even if they aren't strictly whole sentences.